.github/actions/compress_sign_and_upload/action.yml

```diff
@@ -22,22 +22,23 @@ runs:
       shell: bash
 
     - name: Get release version and release package file name
-      id: vars
+      id: get_vars
       shell: bash
       run: |
         package_version=$(jq --raw-output '.version' package.json)
         echo "package_version=${package_version}" >> "$GITHUB_OUTPUT"
         echo "package_file=bson-${package_version}.tgz" >> "$GITHUB_OUTPUT"
-
     - name: Create detached signature
       uses: mongodb-labs/drivers-github-tools/garasign/gpg-sign@v1
       with:
-        filenames: ${{ steps.vars.package_file }}
+        filenames: ${{ steps.get_vars.outputs.package_file }}
```

Comment:
Just for my own understanding, is there a reason we're using `get_vars` instead of `vars` here?

Outside of setting the token, what is the change in logic here?

Reply:
So the issue with the previous PR was just that we didn't have the Github token in the environment?